net/munge/Portfile

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
```

Comment:
You're adding a new file ... then immediately removing a line from said new file. Could you just not add that line in the first place? I think I'd prefer cleaner history than needless commits.